src/main/java/org/primefaces/component/timeline/TimelineBase.java

```diff
@@ -571,6 +572,7 @@ public void setDropScope(String dropScope) {
         getStateHelper().put(PropertyKeys.dropScope, dropScope);
     }
 
+    @Override
     public String getDir() {
         return (String) getStateHelper().eval(PropertyKeys.dir, "ltr");
     }
```

Comment:
You can remove getDir() entirely as RTLAware interface is a default interface that does exactly this right?